### ISSUE TITLE
Disable dragging on a node in editing status

### ISSUE THREAD

**Issue by zheng-wang** [jameskerr/react-arborist#195]
Currently a tree node is still draggable when it is in editing status. This can be seen even in the https://react-arborist.netlify.app/gmail demo. After entering editing status of a node, dragging the input box will drag the node to somewhere else.

Is it possible to disable dragging when the node is in editing status? Just like what we can see in VS Code Explorer pane.

**Comment by jameskerr** [comment]
Yeah that seems reasonable. Would you like to give it shot in a PR?
